package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-planner",
-  "version": "2.0.5",
+  "version": "2.1.0",
```

Comment:
I prefer to maintain the control over the versioning process (for example in this pull requests there are a few bugs so I would avoid publishing a release). So in this case it was preferable to let me merge the pull request and only after completing the process let me release the new version

Reply:
No problem at all!